network-api/networkapi/management/commands/review_app_admin.py

```diff
@@ -19,15 +19,9 @@ def handle(self, *args, **options):
             User.objects.get(username='admin')
             print('super user already exists')
         except ObjectDoesNotExist:
-            password = Faker(
-                'password',
-                length=16,
-                special_chars=True,
-                digits=True,
-                upper_case=True,
-                lower_case=True
-            )
-            User.objects.create_superuser('admin', 'admin@example.com', str(password))
+            fake = Faker()
```

Comment:
just to match the rest of the files where we're directly using faker, let's move this instantiation up to just after line 9 and call it `faker` rather than `fake`.